barretenberg/bbup/bb-versions.json

```
@@ -1,4 +1,5 @@
 {
+  "1.0.0-beta.14": "3.0.0-nightly.20251014",
```

```suggestion
  "1.0.0-beta.14": "3.0.0-nightly.20250916",
```

Barretenberg 3.0.0-nightly.20250916 was the version tested when Noir v1.0.0-beta.14 was cut:
https://github.com/noir-lang/noir/blob/v1.0.0-beta.14/scripts/install_bb.sh